tests/state/test_settings.py

```diff
@@ -166,6 +168,12 @@ def test_validate_no_database_configuration(self):
                 }
             )
 
+    def _test_config_and_credential_synchronization(self):
+        pyiron_key = "PYIRON"
```

Comment:
Looping back to my comment about `PYIRON_DICT_NAME` as a global, now it _really_ smells like that's the case 😉 

Ah, I see now this is actually in a different file. But the comment stands; Right now if we wanted to update its value to "PYIRON_CONFIG_STUFF", we'd need to also update this literal in the test file, so let's define the literal at the class or module level and use it self-consistently here.

Reply:
I declare unresolved! 😉 `pyiron_key` can now be imported instead of redefined

Reply:
Oh, well done :) It already is imported and used at various other places where `'PYIRON'` was before. However I forgot this one for some reason...